pkg/app/server.go

```diff
@@ -118,6 +114,20 @@ func RunKubeStateMetrics(ctx context.Context, opts *options.Options) error {
 			Help: "Timestamp of the last successful configuration reload.",
 		}, []string{"type", "filename"})
 
+	// Register self-metrics to track the state of the cache.
+	crdsAddEventsCounter := promauto.With(ksmMetricsRegistry).NewCounter(prometheus.CounterOpts{
+		Name: "kube_state_metrics_crds_add_events_total",
```

Comment:
Should this be about customresourcestate instead? 
```suggestion
		Name: "kube_state_metrics_crs_add_events_total",
```